dnf-json

```diff
@@ -1,21 +1,30 @@
 #!/usr/bin/python3
 
 import datetime
-import dnf
 import hashlib
-import hawkey
 import json
-import sys
 import tempfile
+import os
+import socket
+import socketserver
+import logging
+from http.server import BaseHTTPRequestHandler
+
+import dnf
+import hawkey
+from systemd.journal import JournalHandler
 
-DNF_ERROR_EXIT_CODE = 10
 
+log = logging.getLogger('dnf-json')
+log.addHandler(JournalHandler())
+log.setLevel(logging.INFO)
+
+DNF_ERROR_EXIT_CODE = 10
 
-def timestamp_to_rfc3339(timestamp):
+def timestamp_to_rfc3339(timestamp, package):
```

Comment:
This change looks right, but how did this work before?

Reply:
If this is a bug fix, maybe making it a separate commit would be the correct thing to do.

Reply:
I the previous version the variable package was in the global scope thus accessible by the function. Since I moved the declaration to inside a class, the function was no longer able to access it, so I added it as a parameter.